hack/verify-gofmt.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+go_files=$( find . -name '*.go' -not -path './vendor/*' -print )
```

Comment:
not `vendor/*`? You don't have a vendor directory for this plugin. Btw why not having it? I'm not a fan of the vendor directory but that's the common practice, I believe even suggested by the golang community.

Reply:
The plugin cannot be used independently. We anyway need to add it to coredns and then build it. When the plugin is added to coredns, the vendor directory of coredns will be updated.

Also it is a norm for all the external plugins to not have any vendor directory. For example: https://github.com/coredns/k8s_event

Reply:
> The plugin cannot be used independently. We anyway need to add it to coredns and then build it.

True, but this script checks the go formatting of the plugin only, not whole coredns. The coredns is cloned into a temp directory before building and is removed after the building.

Reply:
I'll remove the `vendor/*` from the `find` command.